man/tldr.1

```diff
@@ -37,12 +37,12 @@ render a local page for testing purposes
 0 on success, any other positive value otherwise
 .SH SEE ALSO
 The source is available at:
-.URL "https://github.com/tldr-pages/tldr-cpp-client" "" ""
+.URL "https://github.com/tldr-pages/tldr-c-client" "" ""
 .SH REPORTING BUGS
 Report bugs through the Github repository:
-.URL "https://github.com/tldr-pages/tldr-cpp-client" "" "."
+.URL "https://github.com/tldr-pages/tldr-c-client" "" "."
 .SH COPYRIGHT
 The MIT License (MIT)
 
-Copyright (c) 2016 Arvid Gerstmann.
+Copyright (c) 2021 Arvid Gerstmann, the tldr-pages team and contributors.
```

Comment:
```suggestion
Copyright (c) 2014-2021 Arvid Gerstmann, the tldr-pages team and contributors.
```

Copyright notices should cover all years in which the source and changes of it are subject to copyright (basically from the first year to the year of the last change), so in this case 2014-2021.